<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteWindow</class>
 <widget class="QWidget" name="RemoteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>666</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OctoMYâ„¢ - Remote</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/icons.qrc">
    <normaloff>:/icons/remote.svg</normaloff>:/icons/remote.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidgetScreen">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageRunning">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="comboBoxAgent">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>Manage Connections</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/icons.qrc">
              <normaloff>:/icons/pair.svg</normaloff>:/icons/pair.svg</iconset>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxControlLevel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>Connection</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/icons.qrc">
              <normaloff>:/icons/network.svg</normaloff>:/icons/network.svg</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string extracomment="Control individual servos">Actuator</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/icons.qrc">
              <normaloff>:/icons/actuator_control.svg</normaloff>:/icons/actuator_control.svg</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string extracomment="Control pose through playback of animations">Pose</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/icons.qrc">
              <normaloff>:/icons/pose_control.svg</normaloff>:/icons/pose_control.svg</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string extracomment="Control gait dynamically through real-time paramers">Gait</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/icons.qrc">
              <normaloff>:/icons/gait_control.svg</normaloff>:/icons/gait_control.svg</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string extracomment="Leave robot to figure out movement on its own, and let it follow a path on the map">Path</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/icons.qrc">
              <normaloff>:/icons/path_control.svg</normaloff>:/icons/path_control.svg</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Stats</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/icons.qrc">
              <normaloff>:/icons/stats.svg</normaloff>:/icons/stats.svg</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Expression</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/icons.qrc">
              <normaloff>:/icons/mask.svg</normaloff>:/icons/mask.svg</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Speech</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/icons.qrc">
              <normaloff>:/icons/speech.svg</normaloff>:/icons/speech.svg</iconset>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonMenu">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources/icons.qrc">
             <normaloff>:/icons/menu.svg</normaloff>:/icons/menu.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="splitterLog">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="opaqueResize">
          <bool>false</bool>
         </property>
         <property name="handleWidth">
          <number>8</number>
         </property>
         <widget class="QStackedWidget" name="stackedWidgetControl">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="pageControl">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QWidget" name="widgetClientPlaceholder" native="true">
              <layout class="QGridLayout" name="gridLayout_6">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
               <item row="1" column="2" rowspan="2">
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>To control an Agent, please pair with it first. </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButtonStartPairing">
                   <property name="text">
                    <string>Start pairing</string>
                   </property>
                   <property name="icon">
                    <iconset resource="../resources/icons.qrc">
                     <normaloff>:/icons/pair.svg</normaloff>:/icons/pair.svg</iconset>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="2" column="3">
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>137</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="2">
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>19</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="2" column="0">
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>137</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="3" column="2">
                <spacer name="verticalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="LogWidget" name="logScroll">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageConfirmQuit">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>256</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonConfirmQuit">
         <property name="text">
          <string>Confirm Quit</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>256</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pagePairing">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="PairingWizard" name="widgetPairing" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageMyID">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="PortableIDWidget" name="widgetBirthCertificate" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="pushButtonBack_6">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources/icons.qrc">
             <normaloff>:/icons/arrow_left.svg</normaloff>:/icons/arrow_left.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pagePlan">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="pushButtonBack_5">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources/icons.qrc">
             <normaloff>:/icons/arrow_left.svg</normaloff>:/icons/arrow_left.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="PlanEditor" name="widgetPlanEditor" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageDelivery">
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="ControlDeliveryWizard" name="widgetDelivery" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogWidget</class>
   <extends>QPlainTextEdit</extends>
   <header>widgets/LogWidget.hpp</header>
  </customwidget>
  <customwidget>
   <class>PortableIDWidget</class>
   <extends>QWidget</extends>
   <header>widgets/PortableIDWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlanEditor</class>
   <extends>QWidget</extends>
   <header>widgets/planedit/PlanEditor.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PairingWizard</class>
   <extends>QWidget</extends>
   <header>widgets/PairingWizard.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ControlDeliveryWizard</class>
   <extends>QWidget</extends>
   <header>widgets/ControlDeliveryWizard.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
